janusgraph-core/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this file still cause the same problem as reported in https://github.com/JanusGraph/janusgraph/issues/1236? It seems to me that instead of distributing `log4j.properties`, we are now distributing `log4j2.xml`.

To be honest, I don't understand why we have `log4j` configuration in the first place. I thought libraries should only use `slf4j`, and let applications choose their logging backends and provide configurations. I might be totally wrong though.

Reply:
I will remove this. 

I will check where i can remove log4j and where not. 

If you are using janusgraph as library, it would good to let help choose their own logging solution. If you are using janusgraph standalone, it would inconvenient. You would have to setup your own logging solution. 

For tests, it is helpful to be able to get logs. 

I know cql is depend on Cassandra-all which includes logback. Here I'm unable to remove the dependency from the pom.

Reply:
> If you are using janusgraph standalone, it would inconvenient. You would have to setup your own logging solution.

I am not familiar with the logging libraries and their mechanisms, so my question might be dumb. If users use JanusGraph server (so-called janusgraph standalone), is it sufficient to provide `log4j` configurations and dependencies only in `janusgraph-server/pom.xml`?

Reply:
I have to try.